### DISCUSSION TITLE
Dark Mode Issues

### DISCUSSION THREAD

**Discussion by thebaaron42** [burkybang/SuiteAdvanced-Feedback#102] [General]
Hello!

The recent release makes dark mode difficult to use.  The top menu is now black font, and my eyes are struggling to see.  

Also, the analytics section is now unreadable.  Please see screenshots. 

Thank you!
![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/131487/864ede1f-8e88-40c5-b068-d4dfcbaa335b)
![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/131487/d56e290a-8fd6-4df4-99be-24f68df07199)
![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/131487/9948aa4a-3512-4260-a751-c6445e5970a3)

**Comment by burkybang** [comment]
Can you check if you have another extension that adds dark mode to sites? I'm wondering if it's conflicting.

**Comment by thebaaron42** [comment]
Hi there!

I disabled all extensions in Chrome, and turned on only this one.  The issue still persists.

**Reply by burkybang** [reply]
I have created a new issue #104 to keep track of this.

**Comment by thebaaron42** [comment]
Thanks and keep up the good work!

**Comment by eduardRsult** [comment]
![Screenshot 2024-06-26 at 18 21 24](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/122311607/4524ea65-9fdf-4481-b594-3469217fee19)

any fullscreen error thrown

**Comment by HunterJacobs** [comment]
2 Factor Auth
![image](https://github.com/user-attachments/assets/e6978564-cf88-410f-978b-a37337ebe88c)
